Templates/DefaultProject/Template/Gem/enabled_gems.cmake

```diff
@@ -19,7 +19,6 @@ set(ENABLED_GEMS
     ImGui
     LandscapeCanvas
     LyShine
-    Multiplayer
```

Comment:
If we are removing Multiplayer, then we can also consider removing AWSCore gem.

Reply:
Is there a dependency between them? Or for some other reason?

Reply:
If AWSI team is okay with it, so I am. I thought it was only used with Multiplayer.

Reply:
At the moment, I believe AWSCore spams some annoying warning messages on Editor startup.

Reply:
@lmbr-pip thoughts?

Reply:
Log spam issues are known and in the backlog, as @onecent1101 stated they come from AWSClientAuth so we could disable that gem and any other AWS feature gems. AWSCore was deliberately left in this template during O3DE launch (@rainbj for historical context).

At some point the AWS code will need to migrate out of O3DE/O3DE, as vendor gems should not be in the core repro, if/when that plan is accepted IMHO that would be the right time to remove the core gem from the template.